docker/spec/fixtures/docker/registry_tags/git_describe.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "infoblox/grafana-operator",
+  "tags": [
+    "v3.2.0-10-g0208ead",
+    "v3.2.0-10-g17a8c8f",
+    "v3.2.0-10-g2fe2e69",
+    "v3.2.0-10-g3a24f62",
+    "v3.2.0-10-gd2eda66",
+    "v3.2.0-11-gb85335c",
+    "v3.2.0-11-gffe830d",
+    "v3.2.0-12-gbb83bba",
+    "v3.2.0-12-gd0ca143",
+    "v3.2.0-12-gfe00835",
+    "v3.2.0-13-gb4ee0f2",
+    "v3.2.0-164-ge71f8af",
+    "v3.2.0-230-gbabf17f",
+    "v3.2.0-6-g2d0df1d",
+    "v3.2.0-6-g42944f4",
+    "v3.2.0-6-gdcf49b7",
+    "v3.2.0-7-g52cd8b3",
+    "v3.2.0-7-g634e39d",
+    "v3.2.0-7-gf7faf90",
+    "v3.2.0-7-gfad2538",
+    "v3.2.0-7-gff3ea81",
+    "v3.2.0-8-g1648a84",
+    "v3.2.0-8-g1ca2017",
+    "v3.2.0-8-gf4eeefd",
+    "v3.2.0-9-g12202a3",
+    "v3.7.0-13-g02d4c9ef",
+    "v3.7.0-14-g051e1a97",
+    "v3.7.0-15-g5c997230",
+    "v3.7.0-16-ga9d63825",
+    "v3.7.0-16-gd5a70ddc",
+    "v3.7.0-18-ga7fdf65c",
+    "v3.9.0-173-g4d84d01",
+    "v3.9.0-177-ged5bcde",
+    "v3.9.0-181-gbabf17f",
+    "v3.10.0-169-gfe040d3"
```

```suggestion
    "v3.10.0-17-gfefefef",
    "v3.10.0-169-gfe040d3"
```

Idea: verify this bit of the version is being compared as integers and not strings. I think it is - but I wondered for long enough to double check.

```
irb(main):001:0> [17,169].sort
=> [17, 169]
irb(main):002:0> ["17","169"].sort
=> ["169", "17"]
```